build.gradle

```diff
@@ -90,8 +90,8 @@ include {
 		]
 	}
 
-	from('modules/shared/logging.gradle') {
-		slf4jAndLogback '1.7.10', '1.0.13'
+	from('modules/logging.gradle') {
+		slf4jAndLogback '1.7.36', '1.0.13'
```

Comment:
Can you please again include the update to logback from `1.0.13` to `1.2.12`? Thank you